consent-ws/src/main/java/org/broadinstitute/consent/http/service/DatabaseDataSetAPI.java

```diff
@@ -124,53 +170,51 @@ public ParseResult overwrite(File dataSetFile, Integer userId) {
             Collection<Election> dataOwnerOpenElections;
             Collection<String> dataAccessElectionsReferenceId;
             Collection<String> datasetsAssociatedToOpenElections = new HashSet<>();
-            dataOwnerOpenElections = electionDAO.getElectionByTypeAndStatus(ElectionType.DATA_SET.getValue(),ElectionStatus.OPEN.getValue());
-            if(CollectionUtils.isNotEmpty(dataOwnerOpenElections)){
-                dataAccessElectionsReferenceId= dataOwnerOpenElections.stream().map(e -> e.getReferenceId()).collect(Collectors.toSet());
+            dataOwnerOpenElections = electionDAO.getElectionByTypeAndStatus(ElectionType.DATA_SET.getValue(), ElectionStatus.OPEN.getValue());
+            if (CollectionUtils.isNotEmpty(dataOwnerOpenElections)) {
+                dataAccessElectionsReferenceId = dataOwnerOpenElections.stream().map(e -> e.getReferenceId()).collect(Collectors.toSet());
                 datasetsAssociatedToOpenElections = accessAPI.getDatasetsInDARs(dataAccessElectionsReferenceId);
             }
             dataSetDTOList = dsDAO.findDataSets();
             if (userIs(DACUserRoles.ADMIN.getValue(), dacUserId) && dataSetDTOList.size() != 0) {
                 List<Document> accessRequests = accessAPI.describeDataAccessRequests();
-                List<String> dataSetObjectIdList = new ArrayList<>();
-                dataSetDTOList.stream().forEach(dataSet -> {
-                    Map<String,String> dataSetProperties = dataSet.getProperties().stream().collect(Collectors.toMap(DataSetPropertyDTO::getPropertyName,DataSetPropertyDTO::getPropertyValue));
-                    dataSetObjectIdList.add(dataSetProperties.get(DATA_SET_ID));
+                List<Integer> dataSetIdList = new ArrayList<>();
+
+                dataSetDTOList.stream().forEach(dataSet ->{
+                    dataSetIdList.add(dataSet.getDataSetId());
                 });
```

Comment:
`List<Integer> dataSetIdList` could be a one-liner
```suggestion
                List<Integer> dataSetIdList = dataSetDTOList.
                    stream().
                    map(DataSetDTO::getDataSetId).
                    collect(Collectors.toList());
```